src/Teams/Models/Team.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Teams.Models
+{
+    [Table("team")]
+    public partial class Team
+    {
+        public Team()
+        {
+            TeamMembers = new HashSet<TeamMembers>();
+        }
+
+        [Key]
+        public string Id { get; set; }
```

Comment:
Is there any reason to have a string Id? How you will generate unique Id when inserting a new record of this type in the database? Is this will be a Guid? If yes why not to use Guid directly instead of string?